docs/source/contributors.rst

```diff
@@ -0,0 +1,20 @@
+Contributions Welcome!
+======================
+
+**Contributions**
+
+We will support the efforts within the Hyperledger and Decentralized Identity Foundation (DIF) communities to help create innovative distributed ledger technologies and digital identity networks. We believe that creating cross-project and cross-network standardization and interop along with common development frameworks represents the next step in transforming the digital identity space. In particular we support, and look forward to contributing to, the efforts of Hyperledger Fabric, Indy and the newly proposed Aries project along with DIF’s Identity Hub efforts.
```

Comment:
This content was about what SK was planning. It would be better to link to the google doc rather than having it in the repo.

Reply:
Done